project2/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=ZCOOL+QingKe+HuangYou|Khand" rel="stylesheet">
+</head>
+<title>Card Specimen of Spider-Man</title>
+
+<body>
+    <div class="rapper">
+        <figure>
+            <img src="images/Spiderman.jpg" alt="spider-man with black background" width="200px" height="200px">
+        </figure>
+    </div>
+    <div class="context">
+        <h1 class="title" style="width: 360px;">Spider-Man: Into the Spider Verse</h1>
```

Comment:
All styles should be handled in `styles.css` not inline:
```suggestion
        <h1 class="title">Spider-Man: Into the Spider Verse</h1>
```